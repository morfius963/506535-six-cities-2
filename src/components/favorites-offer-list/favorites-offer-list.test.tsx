import * as React from "react";
import * as renderer from "react-test-renderer";
import FavoritesOfferList from "./favorites-offer-list";

describe(`snapshot test`, () => {
  it(`Component correctly renders`, () => {
    const props = {
      city: `Amsterdam`,
      renderOffers: jest.fn()
    };

    const tree = renderer
      .create(<FavoritesOfferList {...props} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
