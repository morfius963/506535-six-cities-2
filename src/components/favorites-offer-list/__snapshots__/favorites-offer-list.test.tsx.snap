// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test Component correctly renders 1`] = `
<li
  className="favorites__locations-items"
>
  <div
    className="favorites__locations locations locations--current"
  >
    <div
      className="locations__item"
    >
      <a
        className="locations__item-link"
        href="#"
      >
        <span>
          Amsterdam
        </span>
      </a>
    </div>
  </div>
  <div
    className="favorites__places"
  />
</li>
`;
